arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1936 @@
+      "type": "object",
+      "properties": {
+        "jobId": {
+          "description": "Service Generated. The job identifier.",
+          "type": "string"
+        },
+        "startTimeUtc": {
+          "format": "date-time",
+          "description": "Service generated. Start time of the Job.",
+          "type": "string"
+        },
+        "endTimeUtc": {
+          "format": "date-time",
+          "description": "Service generated. Represents the time the job stopped processing.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The type of job to execute.",
+          "enum": ["unknown", "export", "import", "backup", "readDeviceProperties", "writeDeviceProperties", "updateDeviceConfiguration", "rebootDevice", "factoryResetDevice", "firmwareUpdate"],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "JobType",
+            "modelAsString": "True"
+          }
+        },
+        "status": {
+          "description": "Service generated. Status of the Job.",
+          "enum": ["unknown", "enqueued", "running", "completed", "failed", "cancelled"],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "JobStatus",
+            "modelAsString": "True"
+          }
+        },
+        "failureReason": {
+          "description": "Service generated. If status == failure, this represents a string containing the reason.",
+          "type": "string"
+        },
+        "statusMessage": {
+          "description": "Status message for the job",
+          "type": "string"
+        },
+        "deviceId": {
+          "description": "The deviceId related to this response.",
+          "type": "string"
+        },
+        "parentJobId": {
+          "description": "The jobId of the parent job, if any.",
+          "type": "string"
+        }
+      }
+    },
+    "IotHubDescriptionListResult": {
+      "description": "Contains the user view of a List Result with a next link",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets the list of",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IotHubDescription"
+          }
+        },
+        "nextLink": {
+          "description": "Gets the next link.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IotHubQuotaMetricInfo": {
+      "description": "Contains properties related to quota metrics for an Iot Hub.",
+      "type": "object",
+      "properties": {
+        "Name": {
+          "description": "Gets or sets the name of the quota metric.",
+          "type": "string"
+        },
+        "CurrentValue": {
+          "format": "int64",
+          "description": "Gets or sets the current value for the quota metric.",
+          "type": "integer"
+        },
+        "MaxValue": {
+          "format": "int64",
```

Comment:
this is a dynamically computed value which may change over time, also not settable by the user.